src/main/java/org/ambraproject/wombat/service/PeerReviewServiceImpl.java

```diff
@@ -134,11 +156,64 @@ String getAllReviewsAsXml(List<Map<String, ?>> reviewLetterItems) throws IOExcep
     // append the acceptance letter
     peerReviewContent += acceptanceLetter;
 
+    peerReviewContent += "<article-received-date>" + articleReceivedDate + "</article-received-date>";
+
     // wrap it in a root node
     peerReviewContent = "<peer-review>" + peerReviewContent + "</peer-review>";
     return peerReviewContent;
   }
 
+  /**
+   * Get received date (aka, original submission date) from article manuscript.
+   * @param itemTable
+   * @return article received date (as a string)
+   */
+  String getArticleReceivedDate(Map<String,?> itemTable) throws IOException {
+
+    Map<String, ?> articleItem = (Map<String, ?>) itemTable.values().stream()
+      .filter(itemObj -> ((Map<String, ?>) itemObj).get("itemType").equals("article"))
+      .collect(MoreCollectors.onlyElement());
+
+    String articleXml = getContentXml(articleItem, "manuscript");
+
+    return getReceivedDate(articleXml);
+  }
+
+  /**
+   * Extract received date from article xml.
+   * @param xml article xml
+   * @return received date.
+   */
+  String getReceivedDate(String xml) throws IOException {
```

Comment:
Given that this is referenced once from a method with a similar name, I recommend inlining this method.